Screens/Turtle/TurtleViewScreen.js

```diff
@@ -71,7 +76,7 @@ export default function TurtleViewScreen({ navigation }) {
             });
     }
 
-    function getSightingByTurtleId(id) {
+    async function getSightingByTurtleId(id) {
```

Comment:
async functions return promises, and you can use await to make the function finish doing that section of code before going on